pkg/client/restclient/request_test.go

```diff
@@ -885,19 +884,15 @@ func TestRequestDo(t *testing.T) {
 		Err     bool
 	}{
 		{
-			Request: &Request{err: errors.New("bail")},
```

Comment:
Is this test case not represented anywhere?  A predetermined error should still exit out.